tests/test_plugin.py

```diff
@@ -88,6 +110,89 @@ def test_not_required_prebuild_plugin_failure(docker_tasker, runner_type, requir
         results = runner.run()
     assert workflow.plugin_failed is required
 
+@pytest.mark.parametrize('success1,success2', [
+    (True, True),
+    (True, False),
+    (False, True),
+    (False, False),
+])
+@pytest.mark.parametrize('stop', [
+    True,
+    False,
+])
```

Comment:
This test doesn't actually make any assertions when `stop` is False, and other tests cover the False case anyway.

It might help readability to remove this parameter. I don't think there is much value in testing what happens when a plugin raises InappropriateBuildStepError but stop_on_success isn't set, since that's not really a valid situation.